### ISSUE TITLE
Kill editors on exit

### ISSUE THREAD

**Issue by phdru** [jlebon/textern#44]
When the browser is closed by the user it kills `textern` (or `textern` terminates itself; I haven't' tested browser crash) but editors are left running. I think `textern` should kill editors on exit.

That's arguable, of course. My editor (`vim`) saves everything when being terminated so I don't care. But I can imagine there are people who would like their editors to left running so they can save before quitting.

Ideally, that should be configurable in the web interface. In that case web extension part must send the configuration option to the native part.

**Comment by cassava** [comment]
While issues such as #60 exist, this "feature" should probably not be merged without an option to disable it.

**Comment by jlebon** [comment]
I agree this should be configurable. And then we can make it opt-in instead of opt-out so we retain the current behaviour.

**Comment by phdru** [comment]
Sorry for late answer — exceptionally busy at the job and gonna leave for 2-weeks vacation.

I agree it should be configurable and off by default. Unfortunately I'm mostly a Python programmer. Can write a little bit of simple JavaScript for sites but not for an extension.

Can cooperate at the Python side if *someone* add a checkbox in the settings and pass its value to the native code.
